index.bs

```diff
@@ -5976,7 +5976,7 @@ partial interface MLGraphBuilder {
 };
 </script>
 
-<div dfn-for="MLGraphBuilder/where(condition, input, other)" dfn-type=argument>
+<div dfn-for="MLGraphBuilder/where(condition, trueValue, falseValue)" dfn-type=argument>
     **Arguments:**
         - <dfn>condition</dfn>: an {{MLOperand}}. The condition tensor.
         - <dfn>trueValue</dfn>: an {{MLOperand}}. The trueValue tensor from which the value is selected when the condition of the corresponding element is set to true.
```

Comment:
Aside: repeating `trueValue` here is a little odd. How about just "The tensor from which..." ?

Reply:
Fine with me.

Reply:
Done in c2001d9479b803ea87837e49cf6e790af9d24552